international-brazilian-journal-of-urology.csl

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" page-range-format="minimal-two" demote-non-dropping-particle="never" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>International Brazilian Journal of Urology</title>
```

Comment:
Okay, let's do that, then. @damnation333, can you make the change?

Personally, I'm not a big fan of journals who adopt acronyms or abbreviations as their official title (I'm looking at you, [BMJ](http://www.bmj.com/)!), but I do think we should follow journal's own naming as much as possible.